fe/fe-core/src/main/java/org/apache/doris/analysis/CreateWorkloadGroupStmt.java

```diff
@@ -69,6 +72,15 @@ public void analyze(Analyzer analyzer) throws UserException {
         if (properties == null || properties.isEmpty()) {
             throw new AnalysisException("Resource group properties can't be null");
         }
+
+        String wgTag = properties.get(WorkloadGroup.TAG);
+        if (!StringUtils.isEmpty(wgTag)) {
```

Comment:
the tag value should be trimed.
And do not use default_tag for workload group without tag.

Reply:
tag name is checked in CreateWorkloadGroupStmt using FeNameFormat.checkCommonName